cmd/network/evpn-vrf.go

```diff
@@ -49,10 +52,6 @@ func CreateVRF() *cobra.Command {
 	cmd.Flags().StringVar(&vtep, "vtep", "", "VTEP IP address")
 	cmd.Flags().StringVar(&addr, "addr", "localhost:50151", "address of OPI gRPC server")
 
-	if err := cmd.MarkFlagRequired("vni"); err != nil {
```

Comment:
can you use `MarkFlagsMutuallyExclusive` or `MarkFlagsRequiredTogether` ?

Reply:
I believe its not possible as LoopbackIpPrefix in only mandatory flag, vni and vtep are optional.